distributor-api-v1/operations.md

```diff
@@ -123,48 +122,149 @@ Preferred initial call used to obtain all static data about distributor configur
+                        "IncludedByDefault": false,
+                        "Name": {
+                            "en-US": "Breakfast"
+                        },
+                        "Ordering": 0,
+                        "Posting": "Once",
                         "RelativePrice": null
                     }
                 ],
+                "TaxEnvironmentCode": "NL",
                 "Telephone": "030 6926666",
-                "TermsAndConditionsUrl": "https://website.com/terms-and-conditions.html"
+                "TermsAndConditionsUrl": {
+                    "en-US": "https://website.com/terms-and-conditions.html"
+                }
             },
-            "Id": "3edbe1b4-6739-40b7-81b3-d369d9469c48",
             "OnlineTravelAgencies": [],
-            "PaymentGatewayEnabled": false,
+            "PaymentCardInput": "NotRequested",
+            "RequiredFields": [],
+            "ServiceId": "c1eec12a-1101-4bg6-ad24-e48f8dlpb9ee"
         }
     ],
     "CurrencyCode": null,
     "CurrencyCodes": [],
+    "DisplayVoucherCode": false,
     "EndDateOffset": null,
     "GtmContainerId": "",
     "IntroVideoUrl": "",
     "LanguageCode": null,
-    "NowUtc": "",
+    "NowUtc": "2020-04-09T07:18:48Z",
     "PrimaryColor": "",
     "StartDateOffset": null,
     "Theme": null,
-    "VoucherCode": null,
-    "DisplayVoucherCode": null,
+    "VoucherCode": ""
 }
 ```
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| `Cities` | array of [City](operations.md#city) | required | Cities supported by hotel. |
-| `CityId` | string | required | ID of default city. |
-| `Configurations` | array of [Configuration](operations.md#configuration) | required | Configurations matching the configuration IDs in request. |
-| `CurrencyCode` | string | optional | Code of default currency accepted by hotel. |
-| `Currencies` | array of [Currency](operations.md#currency) | required | Currencies accepted by hotel. |
-| `EndDateOffset` | string | optional | TBC |
-| `GtmContainerId` | string | optional | Google tag manager indentifier. |
-| `IntroVideoUrl` | string | optional | TBC |
-| `LanguageCode` | string | optional | Code of default language. |
-| `NowUtc` | string | required | Server UTC date and time. |
-| `PrimaryColor` | string | optional | TBC |
-| `StartDateOffset` | string | optional | TBC |
-| `Theme` | string | optional | TBC |
-| `VoucherCode` | string | optional | TBC |
+| `Cities` | array of [City](operations.md#city) | required | Cities supported by the enterprise. |
+| `CityId` | string | required | Unique identifier of the default city. |
+| `Configurations` | array of [Configuration](operations.md#configuration) | required | List of [Configuration](operations.md#configuration)s matching the unique configuration identifiers in the request. |
+| `CurrencyCode` | string | optional | Currency code in ISO format which Distributor should use. |
```

Comment:
ISO-4217 code of the Currency that Distrubot should display prices? Or something like that?